BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpScreenRouter.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SignUpScreenRouter.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 24/07/2023.
+//
+
+import Foundation
+
+protocol SignUpRouterProtocol {
+    func presentAnotherScreen()
+}
+
+final class SignUpScreenRouter: SignUpRouterProtocol {
+    
+    let nextScreen: StoryAssembly
+    let globalCoordinator: IGlobalCoordinator
+    
+    init(nextScreen: StoryAssembly,
+         globalCoordinator: IGlobalCoordinator) {
+        self.nextScreen = nextScreen
+        self.globalCoordinator = globalCoordinator
+     }
+
+    func presentAnotherScreen() {
```

Comment:
It's a bad naming. Name function as screen that will be presented + prefix "present"

Reply:
Function name changed